raiden/utils/http.py

```diff
@@ -37,6 +38,9 @@ def start(self):
         """
         if self.process is None:
             command = self.command
+            if shutil.which(command) is None:
```

Comment:
`command` doesn't need to be a executable or script, but may be a full shell command, to allow easy replacement of configuration/other parameters, and even redirection of synapse logs (i.e. `--local-matrix="STDOUT_SYNAPSE=1 ./.synapse/synapse > mylog.log"` will work as intended). Won't it break with `which`?

Reply:
Nice catch! I amended the commit to ensure we only check for the executable part of `command`.